bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -133,7 +140,10 @@ protected List<NAObject> updateReadings(SecurityApi api) {
                         eventBuffer.put(personId, event);
                     }
                 }
-            });
+                if (oldestKnown == null || event.getTime().isBefore(oldestKnown)) {
```

Comment:
We want to save the most recent event retrieved, so for me you should use `isAfter` rather than `isBefore` ?

Reply:
No, I want to identify the oldest event.

Reply:
I don't understand at all in that case, we don't want to retrieve all the events since the origin at each poll, we just want all those after the last already retrieved.

Reply:
I don't understand at all in that case. 
What you want is to keep trace is the last event retrieved in order to search those most recent than this one at next poll.

Reply:
Yes, maybe you're correct. Let me rethink of this. I may have made a mistake here while building the logic.